text/0000-deprecate-ember-logger.md

```diff
@@ -0,0 +1,265 @@
+- 2018-01-17
+- RFC PR: (leave this empty)
+- Ember Issue: (leave this empty)
+
+# Deprecation of Ember.Logger
+
+## Summary
+
+This RFC provides for the deprecation and eventual removal of Ember.Logger.
+
+## Motivation
+
+There are a variety of features of Ember designed to support old browsers,
+features that are no longer needed. Ember.Logger came into being because
+the browser support for the console was inconsistent. In some browsers,
+like Internet Explorer 9, the console only existed when the developer tools
+panel was open, which caused references to the console object's methods to
+fail to execute. Ember.Logger provided methods that would route to the 
+console when it was available.
+
+With Ember 3.x, Ember no longer supports these older browsers, and hence this
+feature no longer serves a purpose. Removing it will make Ember smaller and 
+lighter.
+
+## Detailed design
+
+### For framework developers
+
+Remove the following 8 direct uses of Ember.Logging from ember.js and 
+ember-data: 
+
+ember-debug:
+*  deprecate (deprecate.js) - Logger.warn
+*  debug (index.js) - Logger.info
+*  warn (warn.js) - Logger.warn
+ember-routing (router.js):
+*  transitioned to - Logger.log
+*  preparing to transition to - Logger.log
+*  intermediate-transitioned to - Logger.log
+ember-testing:
+*  Testing paused (pause_test.js) - Logger.info
+*  Catch-all handler (adapter.js) - Logger.error
+ember-data:
+*  tests\test-helper.js:      Ember.Logger.log(reason, reason.stack);
+(Note: I found one usage in ember-data, but emberobserver claims two more.)
+
+Adjust all test code that redirects logging and sets it back:
+
+ember:
+*    routing: basic_test.js (adjust)
+ember-application - system: 
+*    logging_test.js (remove?)
+*    dependency_injection : default_resolver.test.js (adjust)
+ember-glimmer - integration: 
+*    helpers: log-test.js (remove)
+
+None of the uses of Ember.Logging in ember.js or ember-data are using 
+Ember.debug, which will simplify matters.
+
+### For Framework Users
```

Comment:
The instructions seem for addon developers.

Reply:
Good point - I'll make two sections - not much to say for framework users, though.

Reply:
Agreed, I think the framework users are already covered in the "how we teach this".